wrapper.c

```diff
@@ -10,6 +10,12 @@ void _go_git_populate_remote_cb(git_clone_options *opts)
 	opts->remote_cb = (git_remote_create_cb)remoteCreateCallback;
 }
 
+void _go_git_populate_checkout_cb(git_checkout_options *opts)
```

Comment:
What do you mean?
Something like _go_git_populate_checkout_cbs or _go_git_populate_checkout_notify_and_progress_cbs?